Creating shell with bullet spread

johnsdjustin opened the pull request:
Adding shell scene the instantiates multiple bullets and rotates them to create a spread pattern

johnsdjustin committed:
Creating shell with bullet spread

huttonb commented on bullets/bullets.gd:
```diff
@@ -1,19 +1,19 @@
 
 
 # Called every frame. 'delta' is the elapsed time since the previous frame.
-func _physics_process(delta):
```
should be _physics_process since you're controlling the position, using normal _physics can result in buggy behavior

johnsdjustin replied:
Reverted back to _physics_process

huttonb commented on characters/playerbody.gd:
```diff
@@ -1,9 +1,13 @@
 const JUMP_VELOCITY = -400.0
+var aim_direction = Vector2.RIGHT
+
+func _process(delta):
```
should be in _physics_process

johnsdjustin replied:
Removed lines 7 and 9 because they were unnecessary

johnsdjustin committed:
Changing _process calls to _physics_process calls

- Converting the _process call in bullets.gd to _physics_process
- Removing _process call in shell.gd
- Removing _process call and aim_direction variable in playerbody.gd
